pre_commit/clientlib.py

```diff
@@ -112,6 +112,25 @@ def validate_manifest_main(argv: Optional[Sequence[str]] = None) -> int:
 META = 'meta'
 
 
+class OptionalSensibleRegex:
+    def __init__(self, key: str, check_fn: Any) -> None:
+        self.key = key
+        self.check_fn = check_fn
```

Comment:
```suggestion
class OptionalSensibleRegex(NamedTuple):
    key: str
```

I don't think `check_fn` is needed here

Reply:
i kept the `__init__` function as it caused problems with `apply_default`/`remove_default`:

```console
TypeError: _dct_noop() missing 1 required positional argument: 'dct'
```

Reply:
ah, ok, I ended up adjusting this slightly! no problem!